dockerfiles/bionic/docker-compose.yml

```diff
@@ -0,0 +1,5 @@
+bionic:
+  build: ../../dockerfiles/bionic
+  command: bash -c "cd src && tox -ebionic && dpkg-buildpackage -d -uc -us && mv ../*.deb ../dist/"
```

Comment:
can you do `-e bionic` ?

Does this need to be `bash -c`? I bet you don't need that